common/app/views/fragments/items/elements/facia_cards/title.scala.html

```diff
@@ -17,12 +17,16 @@
 @articleLink(html: Html) = {<a href="@header.url.get(request)" class="fc-item__link" data-link-name="article">@html</a>}
 
 @itemHeader(containerIndex == 0 && itemIndex == 0, header.quoted) {
-    @header.kicker match {
-        case Some(kicker) => {
-            @articleLink{<span class="@kicker.linkClasses.mkString(" ")">@Html(kicker.kickerHtml)</span> @headline()}
-        }
-        case _ => {
-            @articleLink{@headline()}
+    @if(isPaidFor) {
+        @articleLink{<div class="fc-item__kicker">Advertiser content</div> @headline()}
```

Comment:
Do we need to use a `<div>` for the kicker if `isPaidFor`, we normally use a `<span>` otherwise?